tenable/io/v3/platform/groups/api.py

```diff
@@ -0,0 +1,250 @@
+                specified group.
+
+        Example:
+            >>> group_id = '00000000-0000-0000-0000-000000000000'
+            >>> for user in tio.v3.platform.groups.list_users(group_id):
+            ...     pprint(user)
+        '''
+        return self._get(f'{group_id}/users')['users']
+
+    def search(self,
+               **kwargs
+               ) -> Union[CSVChunkIterator,
+                          SearchIterator,
+                          Response]:
+        '''
+        Search and retrieve the user groups based on supported conditions.
+
+        :devportal:`groups: search <groups-list>`
+
+        Args:
+            fields (list):
+                The list of field names to return from the Tenable API.
+                Example:
+                    - ['field1', 'field2']
+            filter (tuple, Dict):
+                A nestable filter object detailing how to filter the results
+                down to the desired subset.
+                Examples:
+                    >>> ('or', ('and', ('test', 'oper', '1'),
+                    ...            ('test', 'oper', '2')
+                    ...     ),
+                    ... 'and', ('test', 'oper', 3)
+                    ... )
+                    >>> {'or': [
+                    ...         {'and': [
+                    ...                 {
+                    ...                     'value': '1',
+                    ...                     'operator': 'oper',
+                    ...                     'property': '1'
+                    ...                 },
+                    ...                 {
+                    ...                     'value': '2',
+                    ...                     'operator': 'oper',
+                    ...                     'property': '2'
+                    ...                 }
+                    ...             ]
+                    ...         }
+                    ...     ],
+                    ... 'and': [
+                    ...         {
+                    ...             'value': '3',
+                    ...             'operator': 'oper',
+                    ...             'property': 3
+                    ...         }
+                    ...     ]
+                    ... }
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth:`tio.v3.platform.definitions.groups()`
+                endpoint to get more details.
+            limit (int):
+                Number of objects to be returned in each request.
+                Default is 200.
+            next (str):
+                The pagination token to use when requesting the next page of
+                results.  This token is presented in the previous response.
+            return_resp (bool):
+                If set to true, will override the default behavior to return
```

Comment:
Correct the docstring to match it with the actual behavior of the parameter.

Reply:
addressed